packages/design-system/src/components/hex/stories/index.js

```diff
@@ -32,6 +31,10 @@ import { Headline } from '../../..';
 export default {
   title: 'DesignSystem/Components/HexInput',
   component: HexInput,
+  args: {
+    label: 'Normal',
+    hint: 'Hint',
+  },
 };
```

Comment:
@BrittanyIRL Do we need this story? It seems to look exactly the same as the `Input` story. I know they're different components but still 🤷 

Maybe it belongs in the `Input` stories as a story there. Ticket for a later time 🤷 

|Hex Input|Input|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/151242240-68a85516-ae84-427d-bf72-ca3dba0ac454.png)|![image](https://user-images.githubusercontent.com/22185279/151242297-b0de9d55-f07b-4b50-8d7a-a5585fc10a46.png)|